### PR TITLE
Chore(docs): document dbt runtime vars

### PR DESCRIPTION
**Pull request by treysp**

### CODE REVIEW

File: docs/integrations/dbt.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ SQLMesh has native support for running dbt projects with its dbt adapter.
 Prepare an existing dbt project to be run by SQLMesh by executing the `sqlmesh init` command *within the dbt project root directory* and with the `dbt` template option:
 
 ```bash
-$ sqlmesh init -t dbt
+$ sqlmesh init <default SQL dialect> -t dbt
```
Comment (crericha) [inline docs/integrations/dbt.md:11]: The default dialect is determined from the profile. dbt users should specify.

Comment (treysp) [reply docs/integrations/dbt.md:11]: Should not specify?

File: docs/integrations/dbt.md [hunk 2]
```diff
@@ -22,6 +24,56 @@ Models **require** a start date for backfilling data through use of the `start`
...
+
+In SQLMesh, these variables are passed via configurations. When you initialize a dbt project with `sqlmesh init`, a file `config.py` is created in your project directory. The file contents are:
+
+```python
+from pathlib import Path
+
+from sqlmesh.dbt.loader import sqlmesh_config
+
+config = sqlmesh_config(Path(__file__).parent)
+```
```
Comment (crericha) [inline docs/integrations/dbt.md:40]: Do we want to include the whole file or just the line that matters? Rationale being that the file created by init could change some over time.

File: docs/integrations/dbt.md (hunk 1 above)
Comment (crericha) [reply docs/integrations/dbt.md:11]: shouldn't, correct.

**Review by crericha — APPROVED** [review]

**Pull request merged**